pages/storage_and_backup/object_storage/s3_asynchronous_replication/guide.en-gb.md

```diff
@@ -0,0 +1,378 @@
+| Prefix                  | An object key name prefix that identifies the object or objects to which the rule applies. To include all objects in a bucket, specify an empty string | No       |
+| ID                      | Each replication rule has a unique ID                                                                                   | Yes      |
+| Filter                  | A filter that identifies the subset of objects to which the replication rule applies. To replicate all objects in the bucket, specify an empty object | Yes      |
+| Destination             | A container for information about the replication destination and its configurations                                     | Yes      |
+| DeleteMarkerReplication | Tells if delete operations should be replicated                                                                         | Yes      |
+| Bucket                  | The destination bucket (to replicate to multiple destinations, you must create multiple replication rules)              | Yes      |
+| And                     | You can apply multiple selection criteria in the filter                                                                 | No       |
+
+### Delete Marker Replication
+
+> [!warning]
+> **IMPORTANT**
+> If you specify a `Filter` in your replication configuration, you **must** also include a `DeleteMarkerReplication` element. If your `Filter` includes a `Tag` element, the `DeleteMarkerReplication` Status **must be set to Disabled**.
+
+
+### Understanding Delete Markers
+
+When a delete object operation is performed on an object in a versioning-enabled bucket, it does not delete the object permanently but it creates a delete marker on the object. This delete marker becomes the latest and current version of the object with a new version id.
+
+A delete marker has the following properties:
+
+- A key and version ID like any other object.
+- It does not have data associated with it, thus it does not retrieve anything from a GET request (you get a 404 error).
+- By default, it is not displayed in the Control Panel UI anymore.
+- The only operation you can use on a delete marker is DELETE, and only the bucket owner can issue such a request.
+
+To permanently delete an object, you have to specify the version-id in your delete object request.
+
+> [!warning]
+> By default, OVHcloud Object Storage does not replicate delete markers nor replicate the permanent deletion to destination buckets. This behavior protects your data from unauthorized or unintentional deletions.
+
+However, you can still replicate delete markers by adding the DeleteMarkerReplication element to your replication configuration rule. DeleteMarkerReplication specifies if delete markers should or should not be replicated (when versioning is enabled, a delete operation is performed on an object it does not actually delete the object but it flags it with a delete marker)
+
+```json
+{
+  "Role": "string",
```

Comment:
feedbacks from dev team: replace "string" in "role" attribute by "arn:aws:iam::<your_project_id>:role/s3-replication"